presto-main/src/main/java/io/prestosql/sql/planner/LocalExecutionPlanner.java

```diff
@@ -1915,13 +1932,17 @@ private PhysicalOperation createLookupJoin(JoinNode node,
                 Optional<Symbol> buildHashSymbol,
                 LocalExecutionPlanContext context)
         {
-            // Plan probe
-            PhysicalOperation probeSource = probeNode.accept(this, context);
+            Optional<DynamicFilter> dynamicFilter = createDynamicFilter(node, probeNode, probeSymbols, buildNode, buildSymbols, context);
```

Comment:
If dynamic filter would not be consumed by table scan then it should not be created. You should check that plan fragment contains table scans (and validate that it's for this particular dynamic filter within join)

Reply:
Done at 39a2eb47da.